package.json

```diff
@@ -7,22 +7,22 @@
     "axios": "^0.18.0",
     "bootstrap": "^4.3.1",
     "font-awesome": "^4.7.0",
-    "moment": "^2.24.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "2.1.8",
+    "react-scripts": "3.0.1",
     "reactstrap": "^7.1.0",
     "storybook-host": "^5.0.3"
   },
   "scripts": {
     "start": "run-p start-dev start-api",
     "start-dev": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "react-scripts build --stats && webpack-bundle-analyzer build/bundle-stats.json -m server",
```

Comment:
Open webpack-bundle-analyzer when the prod build is completed.

Reply:
We have not yet added WBA. Not sure of our need for it